schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -192,7 +192,12 @@
             { "$ref": "#/definitions/genericExtension" },
             { "$ref": "#/definitions/iaaSDiagnostics" },
             { "$ref": "#/definitions/iaaSAntimalware" },
-            { "$ref": "#/definitions/customScriptExtension" }
+            { "$ref": "#/definitions/customScriptExtension" },
+            { "$ref": "#/definitions/customScriptForLinux"},
```

Comment:
Check our Repo home: https://github.com/Azure/azure-resource-manager-schemas. You can add one test case in azure-resource-manager-schemas/tests/2015-08-01/Microsoft.Compute.tests.json, or you can modify an existing one.